src/js/angular/rest/repositories.rest.service.js

```diff
@@ -72,6 +75,19 @@ function RepositoriesRestService($http) {
     }
 
     function validateOntopPropertiesConnection(ontopProperties) {
-        return $http.post(`${REPOSITORIES_ENDPOINT}/test-connection`, ontopProperties);
+        return $http.post(`${REPOSITORIES_ENDPOINT}/ontop/test-connection`, ontopProperties);
+    }
+
+    function getSupportedDriversData() {
+        return $http.get(`${REPOSITORIES_ENDPOINT}/ontop/drivers`);
```

Comment:
Are you sure this service call is OK? As I can see, the rest call fails with status 500 and `{"message":"Request method 'GET' not supported"}`. Better check this!